gearbox.go

```diff
@@ -136,7 +136,7 @@ func (gb *gearbox) Start(address string) error {
 	if err != nil {
 		return err
 	}
-
+	gb.httpServer.Name = address
```

Comment:
According to https://github.com/valyala/fasthttp/blob/32940977fb67c66fa367b7fc8f59cfa0e4f80be7/server.go#L186, this is not the context to save address

Reply:
oh shit.
okay, will take care of it.
The problem is I spent some time trying to get the address the httpserver is listening to but it's inside a not exported array.
I can see it in the debugger but I can't use it.

Reply:
You can save it in gearbox struct if you cannot get it from anywhere

Reply:
@muhammednagy Don't forget to add comments too